themes/base.theme.bash

```diff
@@ -395,6 +395,14 @@ function hg_prompt_vars() {
 }
 
 function node_command_version_prompt() {
+	# Set to 'true' to only show node version in directories with package.json
+	NODE_VERSION_CHECK_PROJECT="${NODE_VERSION_CHECK_PROJECT:-false}"
+
+	# If NODE_VERSION_CHECK_PROJECT is enabled, only show version in Node.js projects
+	if [[ "${NODE_VERSION_CHECK_PROJECT}" == "true" ]] && ! _is_node_project; then
+		return
```

Comment:
Some specify the return code as `return 0`, while some don't. Is there a background? If not, I recommend always specifying the return code explicitly. Bash had some strange behaviors of the `return` statement without exit code inside trap handlers.